### ISSUE TITLE
Feature request: Keep views within screen edges

### ISSUE THREAD

**Issue by duemunk** [nicklockwood/CubeController#1]
It's a really great implementation. I have one small suggestion, and that is to keep the views within the screen edges. This would improve the illusion of the 3D world. 

First off, the max visual height of a view should be no higher than the screen height. Secondly the disappearing view just barely extends beyond the sides of the screen. This might be solved be adjusting the perspective (.m34).

**Comment by nicklockwood** [comment]
This would basically require shifting the cube backwards (away from the camera) at the same time as rotating it. It can be done, but the effect would look a bit peculiar in practice I think.

I might add this as an option.

**Comment by duemunk** [comment]
Exactly! I've seen implementations (in other interface domains) which did keep the max visible height equal to screen height. IMHO, that looked more gullible.
